src/shared/ml-thirdparty-client/models/core/transactions.ts

```diff
@@ -35,3 +54,137 @@ export enum Currency {
   SGD = 'SGD',
   USD = 'USD',
 }
+
+/**
+ * Data model for the complex type GeoCode. 
+ * Indicates the geographic location of a transaction initiation.
+ */
+export interface GeoCode {
+  /**
+   * Latitude of the Party.
+   */
+  latitude: string
+
+  /**
+   * Longitude of the Party.
+   */
+  longitude: string
+}
+
+/**
+ * Data model for the complex type Money.
+ */
+export interface Money {
+  /**
+   * Amount of money.
+   */
+  amount: string
+
+  /**
+   * Currency of the amount.
+   */
+  currency: Currency
+}
+
+/**
+ * Quote of a transaction.
+ */
+export interface Quote {
+  /**
+   * The amount of money that the Payee FSP should receive.
+   */
+  transferAmount: Money
+
+  /**
+   * The amount of Money that the Payee should receive in the end-to-end transaction. 
```

Comment:
Is payeeReceiveAmount equal to transferAmount - payeeFspFee? If so, would be helpful to describe that explicitly here.

Reply:
For this part, I actually just copied the description from the Mojaloop OpenAPI definition. There are quite a lot of things that could affect the `payeeReceiveAmount`, which are the commissions and fees from both Payee and Payer FSPs. If we are about to write the calculation here, I think it would be quite out of scope since the readers will also need to understand how the commissions and fees could cover for each other, how the Payee FSP may hide the fee and get commission from the Payer FSP, etc.  Hence, I feel like it would be better to keep things simple by just writing the comment as it is... What do you think about this?

Reply:
Yes you can just describe it in general, something like payeeReceiveAmount is usually transferAmount less commission and fees. No need to write an essay about it.

Reply:
Okay, added a comment about it